util/topgen.py

```diff
@@ -856,6 +931,7 @@ def _process_top(
 
     # Generate Pinmux
     generate_pinmux(completecfg, out_path)
```

Comment:
Is it still necessary to call `generate_pinmux`?

Reply:
This PR is not done: it needs a few more commits, and this will be dealt with.

Reply:
This is resolved: generate_pinmux itself uses ipgen.